panther_bringup/config/ekf_config.yaml

```diff
@@ -0,0 +1,75 @@
+# Ref: http://docs.ros.org/en/noetic/api/robot_localization/html/state_estimation_nodes.html
+
+frequency: 20
+sensor_timeout: 0.2
+two_d_mode: true
+
+transform_time_offset: 0.0
+transform_timeout: 0.1
+
+map_frame: map
+odom_frame: odom
+base_link_frame: base_link
+world_frame: odom
+publish_tf: true
+publish_acceleration: false
+
+odom0: odom/wheels
+odom0_config: [false,  false,  false,
+               false, false, false,
+               true, true, false,
+               false, false, true,
+               false, false, false]
+odom0_queue_size: 2 
+odom0_nodelay: false
+odom0_differential: false
+odom0_relative: true
+
+imu0: imu/data
+imu0_config: [false, false, false,
+              false, false, false,
+              false, false, false,
+              false, false, true,
+              true, true, false]
+imu0_nodelay: false
+imu0_differential: false
+imu0_relative: true
+imu0_queue_size: 5
+
+imu0_remove_gravitational_acceleration: true
+
+print_diagnostics: false
+debug: false
+debug_out_file: /path/to/debug/file.txt
+
+process_noise_covariance: [1e-4, 0,    0,    0,    0,    0,    0,    0,    0,    0,    0,    0,    0,    0,    0,
+                           0,    1e-4, 0,    0,    0,    0,    0,    0,    0,    0,    0,    0,    0,    0,    0,
+                           0,    0,    1e-4, 0,    0,    0,    0,    0,    0,    0,    0,    0,    0,    0,    0,
+                           0,    0,    0,    5e-3, 0,    0,    0,    0,    0,    0,    0,    0,    0,    0,    0,
+                           0,    0,    0,    0,    5e-3, 0,    0,    0,    0,    0,    0,    0,    0,    0,    0,
+                           0,    0,    0,    0,    0,    5e-3, 0,    0,    0,    0,    0,    0,    0,    0,    0,
+                           0,    0,    0,    0,    0,    0,    2e-6, 0,    0,    0,    0,    0,    0,    0,    0,
+                           0,    0,    0,    0,    0,    0,    0,    2e-6, 0,    0,    0,    0,    0,    0,    0,
+                           0,    0,    0,    0,    0,    0,    0,    0,    2e-6, 0,    0,    0,    0,    0,    0,
+                           0,    0,    0,    0,    0,    0,    0,    0,    0,    5e-4, 0,    0,    0,    0,    0,
+                           0,    0,    0,    0,    0,    0,    0,    0,    0,    0,    5e-4, 0,    0,    0,    0,
+                           0,    0,    0,    0,    0,    0,    0,    0,    0,    0,    0,    5e-4, 0,    0,    0,
+                           0,    0,    0,    0,    0,    0,    0,    0,    0,    0,    0,    0,    5e-3, 0,    0,
+                           0,    0,    0,    0,    0,    0,    0,    0,    0,    0,    0,    0,    0,    5e-3, 0,
+                           0,    0,    0,    0,    0,    0,    0,    0,    0,    0,    0,    0,    0,    0,    5e-3]
```

Comment:
would be nice to add some explanation for these values

Reply:
komentarz sposób obliczania